requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.18.4
```

Comment:
Remove this requirement if it is not used in project.

Reply:
removed